wiki/Beatmapping/Approach_Rate/en.md

```diff
@@ -0,0 +1,49 @@
+---
+tags:
+- AR
+- Reading
+---
+
+# Approach Rate
+
+*(See the [Ranking Criteria](/wiki/Ranking_Criteria/) for suggested AR values.)*
```

Comment:
```suggestion
*For suggested AR values, see: [Ranking Criteria](/wiki/Ranking_Criteria)*
```

There is a for see hatnote format in https://osu.ppy.sh/help/wiki/Article_Styling_Criteria#hatnotes.

Reply:
gotta change some hatnotes on a couple articles then :3